src/Command/ModifyEntity/LineItem/LineItemChangeStateCommand.php

```diff
@@ -173,11 +172,10 @@ static function (string $id): string {
                 );
             }
 
-            /** @var ArrayCollection $lineItems */
             $lineItems = $this->lineItemRepository->findBy([$queryField => $queryValue]);
 
             foreach ($lineItems as $lineItem) {
-                $lineItem->setIsActive($toggle === self::TOGGLE_ACTIVATE);
+                $toggle === self::TOGGLE_ACTIVATE ? $lineItem->enable() : $lineItem->disable();
```

Comment:
Since before the field was named `isActive`, calling constant Activate/Deactivate was ok, but now since the field is called `status`, I would prefer to call enable/disable as the methods on the line item entity suggest. Same for the command itself.

It's not that important or required, but we need to do these kind of relations in our brains

Reply:
I agree, fixed